docs/guides/cluster-resources/full-cluster-backup-and-restore/index.md

```diff
@@ -0,0 +1,660 @@
+spec:
+  addon:
+    name: kubedump-addon
+    tasks:
+      - name: manifest-restore
+        params:
+          IncludeClusterResources: "true"
+          IncludeNamespaces: "kubedb,db"
+          IncludeResources: "*"
+          ExcludeResources: "nodes.metrics.k8s.io,nodes,pods.metrics.k8s.io,endpointslices.discovery.k8s.io" 
+    jobTemplate:
+          spec:
+            serviceAccountName: cluster-resource-reader-writter
+```
+> Note: Azure may restrict certain resources or API groups. If this happens, exclude them using the `ExcludeResources` parameter when applying the `RestoreSession`.
+
+Let's create the `RestoreSession` object we have shown above,
+
+```bash
+$ kubectl apply -f https://github.com/kubestash/docs/raw/{{< param "info.version" >}}/docs/guides/cluster-resources/full-cluster-resource/examples/restoresession.yaml
+restoresession.core.kubestash.com/cluster-resources-restore created
+```
+
+**Verify RestoreSession**: 
+Check if the `RestoreSession` has completed successfully:
+```bash
+$ kubectl get restoresession -n demo
+NAME              REPOSITORY   PHASE       DURATION   AGE
+cluster-restore   azure-repo   Succeeded   31s        54s
+```
+
+
+**Verify `KubeDB` restoration**: 
+
+```bash 
+$ kubectl get pods -n kubedb
+NAME                                            READY   STATUS             RESTARTS        AGE
+kubedb-kubedb-autoscaler-0                      0/1     CrashLoopBackOff   9 (3m44s ago)   24m
+kubedb-kubedb-ops-manager-0                     0/1     CrashLoopBackOff   9 (3m24s ago)   24m
+kubedb-kubedb-provisioner-0                     0/1     CrashLoopBackOff   9 (3m17s ago)   24m
+kubedb-kubedb-webhook-server-656d654875-hm2x5   1/1     Running            1 (24m ago)     24m
+kubedb-petset-7d7f6dccf7-75sjn                  1/1     Running            0               24m
+kubedb-sidekick-944f4df5-nv67b                  1/1     Running            0               24m
+```
+
+>Note: Some of the KubeDB pods may appear in `CrashLoopBackOff` due to license restrictions. After upgrading the `license` for the new cluster, those pods will transition to the `Running` state.
+
+**Verify the `MySQL` database: 
+Check the status of the `MySQL` database: 
+```bash 
+$ kubectl get mysql.kubedb.com  -n db
+NAME       VERSION   STATUS   AGE
+my-mysql   8.1.0              19m
+```
+
+>Note: The `STATUS` field may appear empty initially. After upgrading the license for the new cluster, the database will transition to the `Ready` state.
+
+#### Upgrade the KubeDB License for the New Cluster (aks-2): 
+
+Follow the `KubeDB` [official setup page](https://kubedb.com/docs/latest/setup/install/kubedb/) for getting a license and upgrading the `KubeDB` with that license.
+
+```bash 
+$ export LICENSE_FILE=/home/arnab/Downloads/kubedb-license-aks-2.txt
+$ helm upgrade kubedb oci://ghcr.io/appscode-charts/kubedb \
+        --version <kubedb-version> \
+        --namespace kubedb  \
+        --set-file global.license=$LICENSE_FILE \
```

Comment:
We should add `--reuse-values` flag here.  As users might have other values set.